### ISSUE TITLE
Wow64 Emulation Layer Error

### ISSUE THREAD

**Issue by JosephChristiano** [tdhoward/COMpipe#12]
When running COMpipe.exe on Window Server 2022 Datacenter OS 20348.2402 COMpipe gets an error "This app can't run on your PC".

Event ID 1109 appears in the event log.  "The program or feature COMpipe.exe cannot start or run due to incompatibity with 64-bit versions of Windows. Please contact the software vendor to ask if a 64-bit Windows compatible version is available."

Tried running COMpipe in various capability modes without success.  

Side Note: Spelling error "incompatibity " in the windows error log, pretty sure that's on the Microsoft side.

**Comment by tdhoward** [comment]
@JosephChristiano  This sounds like something that another user ran into, I think: 
https://github.com/tdhoward/COMpipe/issues/8
Recompiling on the host platform seemed to fix it. Let me know if that solves the issue for you. I couldn't reproduce the issue on my Windows 11 installation, so it doesn't seem to help if I recompile it.

**Comment by JosephChristiano** [comment]
Thank you for the guidance Tim!  I spun up a Windows Server 22 VM, installed VS22, and ran the compiler using a Windows Console Application.  

Anyone looking to do this that isn't VS savvy, I tried doing the Windows Desktop App first, which caused errors when I tried to run the app on the other server where VS wasn't installed.  The Console Application worked without an issue, though.

I also noticed that the exe that VS built is only 26k, whereas the one we download is 146k.  Not sure why.

Does all that sound right, Tim?  Thank you again for the program.  I was running ESX Free on my home server with Home Assistant, which requires USB dongles.  Well, thanks to Broadcom I had to dump ESX, but Hyper-V is great; we get no cost (to us) licenses from school.
